samples/bot-formatting-cards/csharp/BotFormattingCards/Resources/adaptivePeoplePersonaCardIcon.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "$schema": "https://adaptivecards.io/schemas/adaptive-card.json",
+  "type": "AdaptiveCard",
+  "version": "1.0.0",
+  "body": [
+    {
+      "type": "TextBlock",
+      "text": "Persona",
+      "weight": "bolder"
+    },
+    {
+      "type": "Component",
+      "name": "graph.microsoft.com/user",
+      "view": "compact",
+      "properties": {
+        "id": "{{Object-ID}}",
+        "displayName": "{{User-Display-Name}}",
+        "userPrincipalName": "{{User-Principal-Name}}"
```

Comment:
Make sure to add the details in Readme for updating these data.

Reply:
Same comments applicable to other Json card as well.

Reply:
Fixed